RedirectApplication/RedirectApplication/Models/UsersAttributes.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace RedirectApplication.Models
+{
+    public class UsersAttributes
```

Comment:
Inconsistent naming of properties. We use CamelCase with public properties in the class